src/app/components/RequiredSteps.tsx

```diff
@@ -0,0 +1,54 @@
+'use client';
+import { Grid } from '@mui/material';
+import { usePathname } from 'next/navigation';
+import React from 'react';
+
+import { AddUserDialog } from './dialogs/AddUserDialog';
+import { useAuth } from '../context/AuthProvider';
+import { apiUser } from '../services/endpoints/user.class';
+
+export const RequiredSteps = ({ children }: { children: React.ReactNode }) => {
+  const { user, isAuthenticated } = useAuth();
+  const pathname = usePathname();
+
+  const [isCreateUserDialogOpened, setIsCreateUserDialogOpened] =
+    React.useState(false);
+  const [hasUser, setHasUser] = React.useState(false);
+  const isAnonymous = !isAuthenticated && pathname === '/viewer';
+
+  console.log(pathname);
```

Comment:
debug code?

Reply:
You are right the mentioned code need to be removed, nice catch